### PR TITLE
Ignore logging in message_bus if logger is not initialised

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

# Problem Statement
- Ignore logging if logger is not initialised

# Design
-  https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/349307984/CORTX+Message+Bus 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [ ] Jira is updated
- [x] Check if the description is clear and explained. 
- [ ] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [x] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/utils/log.py [hunk 1]
```diff
@@ -101,53 +100,62 @@ def _get_logger(syslog_server: str, syslog_port: str, file_name: str, logger_typ
...
-        Log.logger.info(f"[{caller}] {msg}", *args, **kwargs)
+        if Log.logger:
+            caller = inspect.stack()[1][3]
+            Log.logger.info(f"[{caller}] {msg}", *args, **kwargs)
 
     @staticmethod
     def audit(msg, *args, **kwargs):
-        caller = inspect.stack()[1][3]
-        Log.audit_logger.info(f"audit: {msg}", *args, **kwargs)
+        if Log.logger:
```
Comment (sachinpunadikar) [inline py-utils/src/utils/log.py:115]: This code change is not required, auditing should be successful  .. so if logger is not available for that it should fail

File: py-utils/src/utils/log.py [hunk 2]
```diff
@@ -101,53 +100,62 @@ def _get_logger(syslog_server: str, syslog_port: str, file_name: str, logger_typ
...
+            caller = inspect.stack()[1][3]
+            Log.audit_logger.info(f"[{caller}] {msg}", *args, **kwargs)
 
     @staticmethod
     def support_bundle(msg, *args, **kwargs):
-        caller = inspect.stack()[1][3]
-        Log.audit_logger.info(f"support_bundle: {msg}", *args, **kwargs)
+        if Log.logger:
```
Comment (sachinpunadikar) [inline py-utils/src/utils/log.py:121]: Same here

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
work on comments

File: py-utils/src/utils/log.py (hunk 2 above)
Comment (selvakumaar5496) [reply py-utils/src/utils/log.py]: Raised exception as per ujjwals suggestion

File: py-utils/src/utils/log.py (hunk 1 above)
Comment (selvakumaar5496) [reply py-utils/src/utils/log.py]: done. Raised exception as per ujjwals suggestion

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/utils/log.py [hunk 3]
```diff
@@ -101,54 +100,67 @@ def _get_logger(syslog_server: str, syslog_port: str, file_name: str, logger_typ
...
+            raise errors.UtilsError(errors.ERR_NOT_INITIALIZED, "Audit Logger "\
+                "is not initialised")
         caller = inspect.stack()[1][3]
-        Log.audit_logger.info(f"audit: {msg}", *args, **kwargs)
+        Log.audit_logger.info(f"[{caller}] {msg}", *args, **kwargs)
 
     @staticmethod
     def support_bundle(msg, *args, **kwargs):
+        if not Log.audit_logger:
+            raise errors.UtilsError(errors.ERR_NOT_INITIALIZED, "Audit Logger "\
```
Comment (rahul-tripathi-git) [inline py-utils/src/utils/log.py:124]: Should this be just Logger instead of Audit logger?

**Review by rahul-tripathi-git — APPROVED** [review]

**Pull request merged**